main/realmain.cc

```diff
@@ -497,6 +497,9 @@ int realmain(int argc, char *argv[]) {
     if (opts.print.isAutogen()) {
         gs->runningUnderAutogen = true;
     }
+    if (opts.print.isAutogenPrintingSubclassesOrAutoloaderOnly()) {
+        gs->autogenPrintingSubclassesOrAutoloaderOnly = true;
```

Comment:
Should we `ENFORCE(gs->runningUnderAutogen)`?

(If not this condition, are there any conditions we should ENFORCE? The change is void of any new `ENFORCE`s so I'm curious.)

Reply:
I think the `ENFORCE` is a reasonable thing to add.